src/nora/components/NoraTextAreaInput/NoraTextAreaInputExample.module.scss

```diff
@@ -0,0 +1,13 @@
+.LabelClassExample {
+  font-weight: 500;
+  font-size: 11px;
+  line-height: 16px;
+  color: var(--NoraNeutral-400);
+}
+
+.TextClassExample {
+  margin-top: var(--Space-12);
+  font-size: 15px;
```

Comment:
I think we have a compact size for 15 px

Reply:
Where are these font sizes defined? I've having a hard time finding it 😅